perllib/FixMyStreet/Cobrand/Northumberland.pm

```diff
@@ -160,6 +160,44 @@ sub dashboard_export_problems_add_columns {
         nearest_address => 'Nearest address',
     );
 
+    # Replace 'wards' column with 'current_ward' and add 'old_ward'
+    # so the default wards code isn't executed.
+    my @cols = @{ $csv->csv_columns };
+    my @heads = @{ $csv->csv_headers };
+    for (my $i = 0; $i < @cols; $i++) {
+        if ($cols[$i] eq 'wards') {
+            splice @cols, $i, 1, ('current_ward', 'old_ward');
+            splice @heads, $i, 1, ('Current ward', 'Old ward');
+            last;
+        }
+    }
+    $csv->_set_csv_columns(\@cols);
+    $csv->_set_csv_headers(\@heads);
```

Comment:
Is there a more concise way to do this?

Reply:
Hmm sorry, thought we'd have done this before somewhere. That looks fine, though yes, you could make it more concise, by splicing in place, so ditching `@cols` and `@heads` and splicing directly on them, same as `splice_csv_column` is doing, something like:

```
    for (my $i = 0; $i < @{$csv->csv_columns}; $i++) {
        if ($self->csv_columns->[$i] eq 'wards') {
            splice @{$self->csv_columns}, $i, 1, ('current_ward', 'old_ward');
            splice @{$self->csv_headers}, $i, 1, ('Current ward', 'Old ward');
            last;
        }
    }
```